test/uk/gov/hmrc/trusts/controllers/EstateVariationsControllerSpec.scala

```diff
@@ -90,22 +87,21 @@ class VariationsControllerSpec extends BaseSpec with BeforeAndAfter {
     "perform auditing" when {
       "the feature toggle is set to true" in {
 
-        when(mockDesService.variation(any[Variation])(any[HeaderCarrier]))
+        when(mockDesService.estateVariation(any[EstateVariation])(any[HeaderCarrier]))
           .thenReturn(Future.successful(VariationResponse(tvnResponse)))
 
         when(mockConfig.auditingEnabled).thenReturn(true)
-        when(mockConfig.variationsApiSchema).thenReturn(appConfig.variationsApiSchema)
 
-        val requestPayLoad = Json.parse(validVariationsRequestJson)
+        val requestPayLoad = Json.parse(validTrustVariationsRequestJson)
```

Comment:
should this be "validEstateVariationsRequestJson" ?

Reply:
Yes it should be estate. complete.